src/07_slices.py

```diff
@@ -12,26 +12,29 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1])
 
 # Output the second-to-last element: 9
-print()
+print(a[4])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+# print(a[2:]) INCORRECT
+print(a[3:])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+# print(a[2:3]) INCORRECT
+print(a[2:4])
 
 # Output every element except the first one: [4, 1, 7, 9, 6]
-print()
+print(a[1:])
 
 # Output every element except the last one: [2, 4, 1, 7, 9]
-print()
+print(a[:5])
 
 # For string s...
 
 s = "Hello, world!"
 
 # Output just the 8th-12th characters: "world"
-print()
\ No newline at end of file
+print(s[7:12])
+# woohoo I did it!!!!
```

Comment:
Superb job here! Slicing this way can be tricky at first but it looks like you have the hang of it.